2-mandatory/2-oxygen-levels.js

```diff
@@ -11,7 +11,17 @@
     Some string methods that might help you here are .replace() and .substring().
 */
 
-function findSafeOxygenLevel() {}
+function findSafeOxygenLevel(oxygenLevels) {
+let planets=oxygenLevels.filter(oxygenLevel =>oxygenLevel.includes("%"))
+let numberLevels= planets.map(number => parseFloat(number))
```

Comment:
here you could chain together the array methods without having to declare variables with `let`